### ISSUE TITLE
Optional query parameters 

### ISSUE THREAD

**Issue by Krzysztof-Cieslak** [fsprojects/SwaggerProvider#82]
### Description

Optional query parameters generates are not respected - generated functions require providing all query parameters

### Repro steps

https://github.com/Krzysztof-Cieslak/SwaggerProviderOptionalQuery

### Expected behavior

Using optional query parameters should generate functions using optional parameters, or options.

### Schema
```json
{
  "swagger": "2.0",
  "info": {
    "title": "test",
    "version": "0.0"
  },
  "security": [
    {
      "api_key": []
    },
    {
      "seymour_oauth": []
    }
  ],
  "consumes": [
    "application/json",
    "application/x-www-form-urlencoded",
    "application/jwt"
  ],
  "produces": [
    "application/json"
  ],
  "paths": {
    "/accounts/": {
      "get": {
        "summary": "List accounts",
        "parameters": [
          {
            "name": "name",
            "in": "query",
            "required": false,
            "type": "string"
          },
          {
            "name": "date",
            "in": "query",
            "required": false,
            "type": "string"
          }
        ],
        "responses": {
          "200": {
            "description": "Successful"
          }
        }
      }
    }
  }
}
```

**Comment by sergey-tihon** [comment]
Fix released in [v0.8.0](https://www.nuget.org/packages/SwaggerProvider/0.8.0)

**Comment by Krzysztof-Cieslak** [comment]
Thanks a lot! 😍
